packages/mllp-server/src/app.ts

```diff
@@ -0,0 +1,48 @@
+import { Hl7Server } from "@medplum/hl7";
+import type { Logger } from "@metriport/core/util/log";
+import { out } from "@metriport/core/util/log";
+import * as dotenv from "dotenv";
+import * as Sentry from "@sentry/node";
+import { initSentry } from "./sentry";
+
+dotenv.config();
+
+initSentry();
+
+const MLLP_DEFAULT_PORT = 2575;
+
+async function createHl7Server(logger: Logger): Promise<Hl7Server> {
+  const server = new Hl7Server(connection => {
+    logger.log("Connection received");
+
+    connection.addEventListener("message", ({ message }) => {
+      logger.log(
+        `## New Message from ${connection.socket.remoteAddress}:${connection.socket.remotePort} ##`
+      );
+      connection.send(message.buildAck());
+    });
```

Comment:
Should we leave a TODO w/ the ticket number here to flag we want to store the raw messages in S3?

Reply:
Yep, added.